Toolset/palettes/tools/revtools.livecodescript

```diff
@@ -66,6 +66,10 @@ on registerForMessages
    revIDESubscribe "ideExtensionsChanged"
 end registerForMessages
 
+on closeStack
+   revIDEUnsubscribeAll
```

Comment:
Actually I'm going to close this as it should be happening in the frame behavior - I'll have to investigate why.